### PR TITLE
FIX: EditorWindow space not appearing to work (case 1191859).

### PR DESCRIPTION
**Pull request by Rene-Damm**
Fixes https://fogbugz.unity3d.com/f/cases/1191859/

Also added a simple demo for how to use the input system in EditorWindow code.

### CODE REVIEW

File: Packages/com.unity.inputsystem/InputSystem/InputManager.cs [hunk 1]
```diff
@@ -2274,9 +2274,15 @@ private void AddAvailableDevicesThatAreNowRecognized()
...
-                        deviceFlags: m_AvailableDevices[i].isNative ? InputDevice.DeviceFlags.Native : 0);
+                    try
+                    {
+                        AddDevice(m_AvailableDevices[i].description, false,
+                            deviceId: id,
+                            deviceFlags: m_AvailableDevices[i].isNative ? InputDevice.DeviceFlags.Native : 0);
+                    }
+                    catch (Exception)
+                    {
+                    }
```
Comment (Rene-Damm) [inline Packages/com.unity.inputsystem/InputSystem/InputManager.cs:2285]: This is an unrelated tweak to a change I just landed. Some HID devices led to exception spam in the console. The code here should just go and ignore those devices.

**Review by douglas-piconi — APPROVED** [review]
